doc/tutorial.md

```diff
@@ -729,12 +729,80 @@ return line1().then([] {
 });
 ```
 
+The following example is a small seastar function that illustrates the concept.
+
+```cpp
```

Comment:
Up above there's a code snippet that tried to illustrate this. I think it's excessive to have two of them one next to another. I think the best way here is to ... (see next comment)